### DISCUSSION TITLE
UWP应用在TUN模式下是否完全支持？

### DISCUSSION THREAD

**Discussion by RebornForPower** [MetaCubeX/mihomo#1058] [Q&A]
**首先排除使用 UWP 回环限制解除 / UWP工具 / UWP lookback**
类似的讨论在[这里](https://github.com/2dust/v2rayN/issues/2549)有描述，最后使用的方案是用UWP lookback，但似乎在我的电脑上是不行的。

我在 win10 上开启 TUN 模式（ UWP lookback没有开），目前是可以访问 MS Store ，但 Windows 聚焦应用（win自带的锁屏壁纸更换UWP类应用）不正常，具体表现是不能更换锁屏壁纸。我理解 TUN 模式是直接接管系统所有流量的吧，理论上是不需要 UWP lookback 应用也会强制走代理？

当然我知道UWP类应用可以使用UWP lookback来解除相关的限制，但由于TUN本身是一种接管全局流量的一种方式，理论上不需要再开启UWP lookback了，但目前的现象来说似乎TUN不能处理Windows聚焦的问题，所以我想知道这是TUN目前功能不完善，有部分UWP应用可以绕过TUN代理，还是说需要额外的配置？

此外想补充的是 **UWP lookback类工具不能列举所有的UWP类应用** ，比如上述的Windows 聚焦就是个例子，所以使用UWP lookback是有限制的，如果想完全接管系统流量最好的方案还是使用TUN，这也是为什么我首先排除使用UWP lookback的原因。

**Comment by Remonli** [comment]
楼主你解决spotlight不能更新的问题了吗？
